### ISSUE TITLE
Revise bio

### ISSUE THREAD

**Issue by Hallm13** [Hallm13/portfolio#55]

**Comment by Hallm13** [comment]
![about-desktop](https://user-images.githubusercontent.com/1813275/46419537-bd4b9580-c6f3-11e8-8dd1-badd11cb852a.png)

**Comment by Hallm13** [comment]
I've included what I like to fo when I'm not working but not why I want to be a designer. Include civic mission and that you are interested in expanding beyond that. Tell the story of meaningful work

**Comment by Hallm13** [comment]
Grader comments 

I like where you're going with the positivity and personality throughout. These two sentences could be revised, the last one isn't a complete sentence and instead should be a compound sentence separated by a comma like the following:

I can also afford to work on projects I care about, like my company that I started to help municipal governments collaborate with their communities on civic projects.

I would go into some additional detail here about what inspires you to be part of this industry, your experience and skills within your company and how this relates to your work moving forward within design. It's not completely clear what "afford to work on" means here. Are these the types of projects you're looking for? I might rephrase that slightly to describe what about this job interests you.

**Comment by Hallm13** [comment]
1st draft 

Biography
I live in a small town in Texas because I can walk to tennis courts and get ice cream and bring it back to the house before it melts. I can also afford to work on projects I care about. Like my company that I started to help municipal governments collaborate with their communities on civic projects.

I actually became a designer because of that project. I used to work as a product manager building products to help city governments get feedback from citizens. When the company I was working at and other industry leaders started abandoning the space, I decided to take what I’d learned and build a better product—IdeaMapr. I won a grant from the Knight Foundation, hired an engineer, interviewed dozens and dozens of potential users, and got to work learning product design from scratch so I would have some wireframes and visual designs for my engineer to work from.

I made good progress on it over the first couple of years but I could see that I needed to make the product better. I was always interested in design and got to work with a lot of great designers over my years as a product manager, so I decided to get serious about learning design and signed up for an intensive 8-month design program.

Six months into the program, after working on several design projects from UX research to coding prototypes I realized that I want to be a designer full time. I love where I live and I’d like to keep working from here remotely.

Grader comments

Hi Matt! I like where you're going with the details about what got you interested in design. I might focus less on where you live, since being hired will be more based on you and your skills - especially working remotely as that tends to be more competitive. I would focus on your unique passion in this industry and how your experience has prepared you to be a designer, including your time at Bloc. Then I would close with someone that relates to the type of work, team, etc. and encourage them to contact you!
